recipes/configs/eleuther_evaluation.yaml

```diff
@@ -21,6 +21,7 @@ checkpointer:
 tokenizer:
   _component_: torchtune.models.llama2.llama2_tokenizer
   path: /tmp/Llama-2-7b-hf/tokenizer.model
+  max_seq_len: null
```

Comment:
`max_seq_length` is already defined below and is used instead of the one with the tokenizer. To be consistent, let's keep the max_seq_len here in the tokenizer, set it to 4096, remove the other one, and update the eleuther_eval recipe to use the tokenizer's max_seq_len. You simply would need to update the line here: https://github.com/pytorch/torchtune/blob/f9f75bb563ecae371492a9d49da4a9f514c081b3/recipes/eleuther_eval.py#L246

to use the tokenizer max_seq_len: `self._cfg.tokenizer.max_seq_len`